r/R/dplyr-funcs-datetime.R

```diff
@@ -189,6 +189,70 @@ register_bindings_datetime <- function() {
   })
 }
 
+register_bindings_duration <- function() {
+  register_binding("difftime", function(time1,
+                                        time2,
+                                        tz,
```

Comment:
We should dig a little bit more about what | why this is helpful in R? + document it

Reply:
`tz` has an impact only `time1` or `time2` are POSIXlt. Therefore, I think it's safe to deactivate it in this context and reduce some of the complexity.